.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Problem
+<!-- Why does this PR exist? What is the problem you're solving? Answer those questions here. -->
+
+## Solution
+<!-- 
+Explain _what_ you did to solve the problem here, and just as importantly, _why_ you did it that way. _How_ you solved the problem should be clear from the code. Our future selves will rely on this PR and the code to understand this change.
+
+Notably,
+1. If there's something in your code that may be confusing, unclear, or surprising, explain it. And consider whether those clarifications should be comments in your code instead.
+2. If additional steps are necessary after merging the PR, say what they are. 
+3. If you didn't put the code behind a feature flag, say why not, e.g. it can't technically be controlled by FF, etc.
+-->
+
+### Testing
+<!-- Explain how this code was tested here. If you wrote automated tests and no manual testing was conducted or is needed, you can leave this section out.
+-->
```

Comment:
We currently do manual correctness testing as a crutch for not having sufficient automated testing. We're striving to be in a place where manual correctness testing simply isn't needed, because (1) automated tests would give the same or better guarantees (2) automated tests catch regressions, and (3) PR reviewers don't have time to be doing manual correctness testing ... they're likely not running automated tests either, CI is doing that.

Manual acceptance testing is another story, I don't think we have a process for it, but the PR is tied to an issue with acceptance criteria. In theory the acceptance criteria are clear and thorough enough that you can check if the system does what it needs to be doing by checking those criteria.

Reply:
Resolving, feel free to re-open.

Reply:
Completely agree on your points, automated testing should cover everything needed to safely promote to production. I wasn't suggesting reviewers need to also manually test, but rather _should it ever be needed_, there are instructions on how to manually test. But yes, ideally we'd never have to do it.